core/chains/evm/txmgr/broadcaster_test.go

```diff
@@ -1537,17 +1537,20 @@ func TestEthBroadcaster_ProcessUnstartedEthTxs_Errors(t *testing.T) {
 		assert.Contains(t, err.Error(), "insufficient funds for transfer")
 		assert.True(t, retryable)
 
-		// Check it was saved correctly with its attempt
-		etx, err = txStore.FindTxWithAttempts(ctx, etx.ID)
+		// Check it was saved correctly with replaced attempt
+		output, err := txStore.FindTxsByStateAndFromAddresses(ctx, []gethCommon.Address{fromAddress}, txmgrcommon.TxInProgress, testutils.FixtureChainID)
+		updated_etx := output[0]
 		require.NoError(t, err)
-
-		assert.Nil(t, etx.BroadcastAt)
-		assert.Nil(t, etx.InitialBroadcastAt)
-		require.NotNil(t, etx.Sequence)
-		assert.False(t, etx.Error.Valid)
-		assert.Equal(t, txmgrcommon.TxInProgress, etx.State)
-		require.Len(t, etx.TxAttempts, 1)
-		attempt := etx.TxAttempts[0]
+		assert.NotEqual(t, 0, len(output))
+		assert.NotNil(t, updated_etx.CreatedAt)
+		assert.Nil(t, updated_etx.BroadcastAt)
+		assert.Nil(t, updated_etx.InitialBroadcastAt)
+		require.NotNil(t, updated_etx.Sequence)
+		assert.False(t, updated_etx.Error.Valid)
+		assert.Equal(t, txmgrcommon.TxInProgress, updated_etx.State)
+		require.Len(t, updated_etx.TxAttempts, 1)
+
+		attempt := updated_etx.TxAttempts[0]
```

Comment:
There is no behavioral change in the test. If we keep the old one, it will still pass, as they check against the same thing. If we can't test for the added functionality, let's just update the description.